### ISSUE TITLE
Increase size of hidden layers

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#311]
@lissyx ran the TED data set until epoch 290 he found that the training WER decreases as follows
![ted_wer training](https://cloud.githubusercontent.com/assets/12054740/21415966/1dae2298-c80f-11e6-95d9-6b34793a2dbf.png)
However, the validation WER decreases as
![ted_wer validation](https://cloud.githubusercontent.com/assets/12054740/21415977/304388f8-c80f-11e6-8b57-ada351273c4b.png)
What's clear from this is that we are over overfitting the training data. Traditionally, the means to address this is to increase regularization of the model, in our case increase the dropout rate.

What's also clear from this is that the current network geometry is not sufficient to reach our goal of 10% WER on the test data, as even upon overfitting the training data the WER on the training data is 20%.

So, if we were to take the traditional route of increasing the dropout rate to address overfitting this would cause the WER of the training and validation sets to be closer. However, due to the decreased capacity of the model, it would cause the WER of training, validation, and test to be far from the target of 10%.

So another route to address this is to increase the model's capacity. This should lower the WER of the training data set. Hopefully, this will also lower the validation WER.

In support of this hypothesis the original DeepSpeech paper trained on the Switchboard corpus (SWB), which is about the same size as the TED corpus, and used a model with a capacity greater than ours. Our model has hidden layers of 494 neurons. However the original DeepSpeech paper used hidden layers of 2048 neurons for a similar task

<img width="616" alt="swb" src="https://cloud.githubusercontent.com/assets/12054740/21416202/3f9fe9c0-c811-11e6-8e90-c7782ea825f9.png">

Thus the goal of this issue is to increase the number of neurons in our hidden layers to 2048 while also adjusting the batch size to compensate for the limited GPU memory.
